ios-training/Controller/ViewController.swift

```diff
@@ -40,23 +42,26 @@ class ViewController: UIViewController {
 }
 
 extension ViewController: WeatherDelegate {
-    func loadWeather(_ result: Result<String, YumemiWeatherError>) {
+    func loadWeather(_ result: Result<Response, YumemiWeatherError>) {
         Task {
             switch result {
-            case .success(let string):
-                switch string {
+            case .success(let response):
+                weatherImageView.image = UIImage(named: response.weatherCondition.rawValue)
+                
+                switch response.weatherCondition.rawValue {
```

Comment:
enumを使ってる場合は、型のメリットを活かすために、なるべくrawValueせずにenumのままとすることが望ましいです。

```suggestion
                switch response.weatherCondition{
                case .sunny:
```